example_xla.py

```diff
@@ -96,9 +97,11 @@ def main(
         ckpt_dir, tokenizer_path, rank, world_size, max_seq_len, max_batch_size, dim, n_layers, n_heads
     )
 
-    prompts = [
+    prompts = [generator.tokenizer.decode(list(range(prompt_len)))]
+    print(prompts)
```

Comment:
is this for debug only?

Reply:
Yeah, I'll remove the print statement once all other comments are settled.